### ISSUE TITLE
[a11y] - printing - missing aria-label for <form> Element

### ISSUE THREAD

**Issue by transkamp** [open-pioneer/trails-openlayers-base-packages#442]
### Issue description

open https://open-pioneer.github.io/trails-demo/openlayers-base-packages/dev/samples/showcase/?demo=printing

the form element needs an extra aria-label for better screen reader readability. 
![Image](https://github.com/user-attachments/assets/366ad803-fde0-4e0b-9099-7a0961eb4482)

### Validations

- [x] Read and follow our [code of conduct](https://github.com/open-pioneer/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [contributing guidelines](https://github.com/open-pioneer/trails-starter/blob/main/docs/Contributing.md)
- [x] Check that there isn't already a similar issue
